.github/workflows/test.yaml

```diff
@@ -0,0 +1,25 @@
+name: Run tests
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # The list of python versions should be the same as in the tox.ini file.
```

Comment:
nit: "... should be according to the supported Python versions in the pyproject.toml"